lib/srv/desktop/rdp/rdpclient/client.go

```diff
@@ -72,7 +72,9 @@ import "C"
 
 import (
 	"context"
+	"encoding/binary"
 	"fmt"
+	uuidp "github.com/google/uuid"
```

Comment:
Why `uuidp` and not `uuid`?

Reply:
There was a clash between field name and package name, now it can be simply `uuid`